src/Keboola/Console/Command/MassDeleteProjectWorkspaces.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Console\Command;
+
+use Keboola\Csv\CsvFile;
+use Keboola\Sandboxes\Api\Client as SandboxesClient;
+use Keboola\Sandboxes\Api\Sandbox;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException as StorageApiClientException;
+use Keboola\StorageApi\Components;
+use Keboola\StorageApi\Workspaces;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+class MassDeleteProjectWorkspaces extends Command
+{
+    private const ARGUMENT_STACK_SUFFIX = 'stack-suffix';
+    private const ARGUMENT_SOURCE_FILE = 'source-file';
+    private const OPTION_FORCE = 'force';
+
+    protected function configure()
+    {
+        $this
+            ->setName('manage:mass-delete-project-workspaces')
+            ->setDescription('Mass project enable dynamic backends')
+            ->addArgument(self::ARGUMENT_STACK_SUFFIX, InputArgument::REQUIRED, 'stack suffix "keboola.com, eu-central-1.keboola.com"')
+            ->addArgument(self::ARGUMENT_SOURCE_FILE, InputArgument::REQUIRED, 'Source csv with "prjId,workspaceSchema" columns')
+            ->addOption(self::OPTION_FORCE, 'f', InputOption::VALUE_NONE, 'Write changes');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $connectionUrl = 'https://connection.' . $input->getArgument(self::ARGUMENT_STACK_SUFFIX);
+        $sandboxesUrl = 'https://sandboxes.' . $input->getArgument(self::ARGUMENT_STACK_SUFFIX);
+        $sourceFile = $input->getArgument(self::ARGUMENT_SOURCE_FILE);
+        $output->writeln(sprintf('Fetching projects from "%s"', $sourceFile));
+        $force = $input->getOption(self::OPTION_FORCE);
+
+        // map by project id
+        /**
+         * @var array{
+         *     string,
+         *     string[],
+         * } $map
+         */
+        $map = [];
+        $csv = new CsvFile($sourceFile);
+        foreach ($csv as $i => $line) {
+            if ($i === 0) {
+                // skip header
+                continue;
+            }
+            if (array_key_exists($line[0], $map)) {
+                $map[$line[0]][] = $line[1];
+            } else {
+                $map[$line[0]] = [$line[1]];
+            }
+        }
```

Comment:
Tohle dost spoliha na strukturu toho file (je tam header, je to seznam workspaces ve formatu `WORKSPACE_xy`). Pokud by ten command mel zustat, nestalo by za to udelat aspon simple validaci, ze to nekdo omylem nepusti s nejakou blbosti?

Reply:
přidal sem validaci